lexicon.html

```diff
@@ -185,6 +185,14 @@ <h2>Eve Terms</h2>
       a person (you) flying a kite (them): the kite does not catch up to the
       flyer, but rather stays at a fixed distance from them.
     </p>
+    <p>
+      <a name="krab" href="#krab">Krab</a>:
+      To engage in PVE content for ISK-making purposes, such as ratting,
+      mining, missioning, etc. Also used as a noun, to denote a player or
+      alliance whose primary activity is krabbing; or as an adjective, to
+      denote a system or ship fitting whose primary purpose is to enable
+      krabbing. Approximately synonymous with "carebear" in other games.
+    </p>
```

Comment:
Suggest rewording second sentence to "to denote a player or **group** whose primary activity is krabbing"

It's probably worth noting here that this is often pejorative.